src/vis/density/utils.tsx

```diff
@@ -0,0 +1,107 @@
+import { merge } from 'lodash';
+import {
+  EColumnTypes,
+  ESupportedPlotlyVis,
+  IVisConfig,
+  VisColumn,
+  IDensityConfig,
+  VisNumericalValue,
+  VisCategoricalValue,
+  ColumnInfo,
+  VisNumericalColumn,
+  EHexbinOptions,
+  EScatterSelectSettings,
+} from '../interfaces';
+import { resolveColumnValues, resolveSingleColumn } from '../general/layoutUtils';
+
+export function isDensity(s: IVisConfig): s is IDensityConfig {
+  return s.type === ESupportedPlotlyVis.DENSITY;
+}
+
+const defaultConfig: IDensityConfig = {
+  type: ESupportedPlotlyVis.DENSITY,
+  numColumnsSelected: [],
+  color: null,
+  isOpacityScale: true,
+  isSizeScale: false,
+  hexRadius: 16,
+  dragMode: EScatterSelectSettings.RECTANGLE,
+  hexbinOptions: EHexbinOptions.COLOR,
+};
```

Comment:
This seems to be equal to the code in Vis.tsx. Maybe export it here and use it in Vis.tsx?

Reply:
Done